src/components/ActionModal.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable max-len */
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {
+  StyleSheet, Text, View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import Modal from 'react-native-modal';
+import NewHathorButton from './NewHathorButton';
+
+const FeedbackModal = (props) => (
```

Comment:
Why did you call this function `FeedbackModal` if the file name is another one?

Reply:
Did it wrong.